python/src/aiconfig/editor/client/src/shared/types.ts

```diff
@@ -66,6 +66,7 @@ export function aiConfigToClientConfig(aiconfig: AIConfig): ClientAIConfig {
 export type LogEvent =
   | "ADD_PROMPT"
   | "CLEAR_OUTPUTS"
+  | "DELETE_OUTPUT"
```

Comment:
nit: Would like to use wording "CLEAR_OUTPUT" to match with our UI, but again not a blocker or a big deal

Reply:
i think maybe as a bigger refactor of the api? i made it clear output at first but then found that all the apis reference delete output so wanted to stay consistent there.